src/redturtle/prenotazioni/restapi/serializers/adapters/prenotazioni_folder.py

```diff
@@ -9,18 +8,20 @@
 from redturtle.prenotazioni.interfaces import IRedturtlePrenotazioniLayer
 
 
+# Did not delete this serializer so as is being used by design.plone.ioprenoto
```

Comment:
update design.plone.ioprenoto, do not let garbage here (there is enough already).

Reply:
Done